python_examples/sim_speeds/simple_parameters.json

```diff
@@ -0,0 +1,407 @@
+{
```

Comment:
If this is the same parameters file as for the basic simulation, can we not just read that in?